registry.json

```diff
@@ -5300,5 +5300,20 @@
     "telegram": "Clover_Nodes",
     "operatorName": "CloverNodes",
     "github": "clovernodes"
+  },
+  {
+    "ticker": "VICIS",
+    "poolId": "8881495966f1b73427233aa761c0fd04adfd91b3599a012d7d845472",
+    "social": {
+      "website": "https://vicispool.com",
+      "twitter": "VICISpool",
+      "facebook": "",
+      "discord": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "VICISpool",
```

Comment:
Please can you change this to the telegram account of an individual and not a group or channel? Thanks